controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -154,6 +159,25 @@ func (r *NodeNetworkConfigurationPolicyReconciler) initializeEnactment(policy nm
 	})
 }
 
+func (r *NodeNetworkConfigurationPolicyReconciler) maxUnavailableNodesCount(policy *nmstatev1beta1.NodeNetworkConfigurationPolicy) (int, error) {
+	nmstateNodes, err := policyconditions.NodesRunningNmstate(r.Client)
+	if err != nil {
+		return 0, err
+	}
+	intOrPercent := intstr.FromInt(DEFAULT_MAXUNAVAILABLE)
+	if policy.Spec.MaxUnavailable != nil {
+		intOrPercent = *policy.Spec.MaxUnavailable
+	}
+	maxUnavailable, err := intstr.GetValueFromIntOrPercent(&intOrPercent, len(nmstateNodes), true)
```

Comment:
Looks like we have to use `GetScaledValueFromIntOrPercent` since `GetValueFromIntOrPercent` is being deprecated, 
https://github.com/kubernetes/apimachinery/blob/master/pkg/util/intstr/intstr.go#L141-L168

Reply:
Right! I didn't see that because I had old vendor packages. Done